src/db/pooled_resource_factor_junctions_tables_vco.tsv

```diff
@@ -2,3 +2,4 @@ resource	factor	unique_id	minimum	maximum	specific_faction_set
 #pooled_resource_factor_junctions_tables;2;db/pooled_resource_factor_junctions_tables/vco					
 emp_progress	vco	wh2_dlc13_resource_factor_vco	0	2147483647	
 brt_chivalry	vco	chivalry_vco	-2147483647	2147483647	
+skaven_food	vco	skaven_food_victory	-2147483647	2147483647	
```

Comment:
W: Again, for the sake of coherence: `skv_food` and `food_vco`/`skv_food_vco`/`skaven_food_vco`.

Reply:
These are named as CA name them - I copied their naming pattern. Happy to change them.

Reply:
Re: `skaven_food` vs `skv_food` it's already coded as `skaven_food` by CA, despite the rest of the Skaven resources being prefixed with `skv_`.

![image](https://user-images.githubusercontent.com/46346590/191860143-04d9d827-69ba-4fdc-8eaa-14a59ea5057b.png)

Reply:
I've updated the rest of the names to be consistent with previous naming patterns however.